extension/src/Runner/runnerResultStream.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import * as events from "events";
+import { Readable } from "stream";
+
+export class RunnerResultStream extends events.EventEmitter implements IRunnerResultStreamEvent {

```

Comment:
What does this stream really do?

Reply:
the stream contains running result. Because I'd like to design the running behavior in an asynchronous way, therefore I need to expose a stream as run result so that it can update test status while running